File: TransCoder_tokenized_test_set_functions/cpp/MAXIMUM_LENGTH_PREFIX_ONE_STRING_OCCURS_SUBSEQUENCE_ANOTHER.cpp
int maxPrefix ( char s [ ], char t [ ] ) {
  int count = 0;
  for ( int i = 0;
  i < strlen ( t );
  i ++ ) {
    if ( count == strlen ( s ) ) break;
    if ( t [ i ] == s [ count ] ) count ++;
  }
  return count;
}

File: TransCoder_tokenized_test_set_functions/java/MAXIMUM_LENGTH_PREFIX_ONE_STRING_OCCURS_SUBSEQUENCE_ANOTHER.java
static int maxPrefix ( String s, String t ) {
  int count = 0;
  for ( int i = 0;
  i < t . length ( );
  i ++ ) {
    if ( count == t . length ( ) ) break;
    if ( t . charAt ( i ) == s . charAt ( count ) ) count ++;
  }
  return count;
}